Composer/packages/client/src/components/CreationFlow/v2/CreateBot.tsx

```diff
@@ -199,17 +199,22 @@ export function CreateBotV2(props: CreateBotProps) {
 
   useEffect(() => {
     const itemKey = selectedProgLang.props.itemKey;
+    let newTemplates: BotTemplate[] = [];
     if (itemKey === csharpFeedKey) {
-      const newTemplates = templates.filter((template) => {
+      newTemplates = templates.filter((template) => {
         return template.dotnetSupport;
       });
-      setDisplayedTemplates(newTemplates);
     } else if (itemKey === nodeFeedKey) {
-      const newTemplates = templates.filter((template) => {
+      newTemplates = templates.filter((template) => {
         return template.nodeSupport;
       });
-      setDisplayedTemplates(newTemplates);
     }
+    if (creationFlowType === 'Skill') {
+      newTemplates = newTemplates.filter((template) => {
+        return !template.isMultiBotTemplate;
+      });
+    }
+    setDisplayedTemplates(newTemplates);
```

Comment:
Is it possible to get down here without any of those three alternatives being true? That would cause `newTemplates` to stay blank, and I don't know if that's a problem or not.

Reply:
Not possible, `ItemKey` aka `selectedProgLang.props.itemKey` always has a value so one of the first two conditionals is always hit